app/cypress/e2e/0-ndr-core-tests/gp_user_workflows/download_lloyd_george_workflow.cy.js

```diff
@@ -81,6 +211,42 @@ describe('GP Workflow: View Lloyd George record', () => {
             },
         );
 
+        it(
+            'should display an alert if user click "Download selected files" without selecting anything',
+            { tags: 'regression' },
+            () => {
+                beforeEachConfiguration(Roles.GP_ADMIN);
+                proceedToDownloadSelectionPage();
+
+                // Select documents page
+                cy.title().should(
+                    'eq',
+                    'Download the Lloyd George record for this patient - Digital Lloyd George records',
+                );
+                cy.wait('@searchDocumentReferences');
+
+                cy.getByTestId('download-selected-files-btn').should('exist');
+                cy.getByTestId('download-selected-files-btn').click();
+
+                cy.title().should(
+                    'not.equal',
+                    'Downloading documents - Digital Lloyd George records',
+                );
+                cy.title().should(
+                    'equal',
+                    'Download the Lloyd George record for this patient - Digital Lloyd George records',
+                );
+
+                cy.get('#error-box-summary').should('be.visible');
+                cy.get('#error-box-summary').should('have.text', 'There is a problem');
+                cy.get('.nhsuk-error-summary__body').should('be.visible');
+                cy.get('.nhsuk-error-summary__body').should(
```

Comment:
dataId recommendation from above applies here.
Give the specific text references a unique data-id